### PR TITLE
MOBILE-2163: WBanner title has bad padding if no left icon.

### PR DESCRIPTION
**Pull request by jordanross-wf**
Description
---
A WBanner without the left icon has bad title padding.

What Was Changed
---
Fixed padding issue when there is no title icon in a WBanner

Testing
---
* Run example app, verify the bottom banner example (that no longer has a title icon) shows correct padding for the title

---

Please Review: @Workiva/mobile

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
Number of Findings: 0

File: Example/WMobileKitExample/ModalViewExamplesVC.swift [hunk 1]
```diff
@@ -416,7 +416,7 @@ public class ModalViewExamplesVC: WSideMenuContentVC {
 
         bottomBanner = WBannerView(rootView: view,
             titleMessage: "Bottom Banner Title",
-            titleIcon: UIImage(named: "alert"),
+            titleIcon: nil,
```
Comment (jeffscaturro-wf) [inline Example/WMobileKitExample/ModalViewExamplesVC.swift]: Nice!

File: Source/WBanner.swift [hunk 2]
```diff
@@ -230,7 +230,7 @@ public class WBannerView: UIView {
             if (titleIconImageView.image != nil) {
                 make.left.equalTo(titleIconImageView.snp_right).offset(6)
             } else {
-                make.left.equalTo(self).offset(-BANNER_DEFAULT_RIGHT_PADDING)
+                make.left.equalTo(self).offset(BANNER_DEFAULT_RIGHT_PADDING)
```
Comment (jeffscaturro-wf) [inline Source/WBanner.swift]: This is awesome!

**Comment by jeffscaturro-wf** [comment]
+10

- [ ] Unit test created or updated. - n/a
- [X] Acceptance criteria met.
- [X] Edge cases tested.
